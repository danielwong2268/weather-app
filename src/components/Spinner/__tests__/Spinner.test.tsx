import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Spinner from '..';

describe('<Spinner />', () => {
  it('Renders a div with className "lds-ripple" and two children divs', () => {
    const renderer = ShallowRenderer.createRenderer();
    renderer.render(<Spinner />);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});