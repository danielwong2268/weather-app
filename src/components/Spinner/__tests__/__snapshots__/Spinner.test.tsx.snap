// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Renders a div with className "lds-ripple" and two children divs 1`] = `
<div
  className="css-13jsfnt"
>
  <div
    className="lds-ripple"
  >
    <div />
    <div />
  </div>
</div>
`;
